lib/tinybucket/model/commit.rb

```diff
@@ -4,7 +4,8 @@ module Tinybucket
   module Model
     # Commit Resource
     #
-    # @see https://confluence.atlassian.com/bitbucket/commits-or-commit-resource-389775478.html
+    # @see https://developer.atlassian.com/bitbucket/api/2/reference/resource/repositories/%7Busername%7D/%7Brepo_slug%7D/commit
+    #      https://developer.atlassian.com/bitbucket/api/2/reference/resource/repositories/%7Busername%7D/%7Brepo_slug%7D/commits
```

Comment:
This yard comment shows the `commits URL` text which linked the `commit URL` in generated document.

How about this code if you want to create two links?

```diff
$ git diff lib/tinybucket/model/commit.rb
diff --git a/lib/tinybucket/model/commit.rb b/lib/tinybucket/model/commit.rb
index 8a251a3..fec4521 100644
--- a/lib/tinybucket/model/commit.rb
+++ b/lib/tinybucket/model/commit.rb
@@ -5,8 +5,9 @@ module Tinybucket
     # Commit Resource
     #
     # @see https://developer.atlassian.com/bitbucket/api/2/reference/resource/repositories/%7Busername%7D/%7Brepo_slug%7D/commit
-    #      https://developer.atlassian.com/bitbucket/api/2/reference/resource/repositories/%7Busername%7D/%7Brepo_slug%7D/commits
-    #   Commit Resource - Bitbucket Cloud REST API document
+    #   Commit resource - Bitbucket Cloud REST API document
+    # @see https://developer.atlassian.com/bitbucket/api/2/reference/resource/repositories/%7Busername%7D/%7Brepo_slug%7D/commits
+    #   Commits resource - Bitbucket Cloud REST API document
     #
     # @!attribute [rw] hash
     #   @return [String]
```